x-pack/plugins/security_solution/public/alerts/pages/detection_engine/rules/details/index.tsx

```diff
@@ -385,6 +392,12 @@ export const RuleDetailsPageComponent: FC<PropsFromRedux> = ({
                         )}
                       </>
                     )}
+                    {ruleDetailTab === RuleDetailTabs.exceptions && (
+                      <ExceptionsViewer
+                        commentsAccordionId={'ruleDetailsTabExceptions'}
+                        exceptionLists={[]}
+                      />
+                    )}
```

Comment:
Good call on leveraging the existing tabs here so we can start getting things on the screen before having to do the  Rule Details refactoring (https://github.com/elastic/kibana/issues/65937). 🙌  🙂

Reply:
Yea, I figured we could sneak this in there now. It didn't disrupt the flow much.

Reply:
As an aside, we're going to need to expose the `Rule Details` / Exceptions` tabs (and probably these sub-tabs) as part of the URL so we can deep-link to them. Nothing to do as part of this PR, but just mentioning.

Reply:
Good to keep in mind - thanks!